test/utils.js

```diff
@@ -18,6 +18,22 @@ test('merge()', function (t) {
     var nestedArrays = utils.merge({ foo: ['baz'] }, { foo: ['bar', 'xyzzy'] });
     t.deepEqual(nestedArrays, { foo: ['baz', 'bar', 'xyzzy'] });
 
+    if (typeof Object.defineProperty === 'function') {
```

Comment:
instead, `t.test(msg, { skip: typeof Object.defineProperty !== 'function' }, function (st) {` should work

Reply:
> @jchook is this still something you're interested in?

Sure I'll take a look at your comments.

How should I handle getting the branch up-to-date? Rebase? New branch?